### PR TITLE
Remove MZBD-73 Remove Amazon engines

### PR DESCRIPTION
**Pull request by mkaply**
## :scroll: Tickets
https://mozilla-hub.atlassian.net/browse/MZBD-73

## :bulb: Description
Removing Amazon except for Japan

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [X] Wrote unit tests and/or ensured the tests suite is passing
- [X] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [X] If needed, I updated documentation / comments for complex code and public methods
- [X] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Review by nbhasin2 — APPROVED** [review]

**Comment by isabelrios** [comment]
There is an automated UI Test failure due to the changes in this PR: testSearchEngine() that is blocking the merge.

@dragosb01 @clarmso could you please take a look to unblock this PR?

**Comment by dragosb01** [comment]
@mkaply you need to delete line 196 from SearchTests class and the test will pass.

**Comment by mkaply** [comment]
Thanks for the pointer, I had missed that.

**Comment by clarmso** [comment]
Let me try to run a Bitrise pipeline.

File: focus-ios/focus-ios-tests/XCUITest/SearchProviderTest.swift
Comment (clarmso) [inline focus-ios/focus-ios-tests/XCUITest/SearchProviderTest.swift:23]: Note for self: Update test case.

File: focus-ios/focus-ios-tests/XCUITest/SearchProviderTest.swift
Comment (clarmso) [inline focus-ios/focus-ios-tests/XCUITest/SearchProviderTest.swift:24]: Note for self: Update test case.

**Review by isabelrios — APPROVED** [review]

**Pull request merged**
